docs/updating/upgrading_to_v3.md

```diff
@@ -0,0 +1,570 @@
+    BEGIN;
+    ALTER TABLE ezdatebasedpublisher_scheduled_version RENAME COLUMN publication_date TO date;
+    ALTER TABLE ezdatebasedpublisher_scheduled_version ADD COLUMN action VARCHAR(32);
+    UPDATE ezdatebasedpublisher_scheduled_version SET action = 'publish';
+    ALTER TABLE ezdatebasedpublisher_scheduled_version ALTER COLUMN action SET NOT NULL ;
+    COMMIT;
+    ```
+
+### Site Factory
+
+For production, it is recommended to create the `ezsite` and `ezsite_public_access` tables manually importing their schemas definition:
+
+??? "Table structure for table `ezsite` and `ezsite_public_access` (MySQL)"
+
+    ```sql
+    DROP TABLE IF EXISTS `ezsite`;
+    
+    CREATE TABLE `ezsite` (
+      `id` int(11) NOT NULL AUTO_INCREMENT,
+      `name` varchar(255) COLLATE utf8mb4_unicode_520_ci NOT NULL DEFAULT '',
+      `created` int(11) NOT NULL,
+      PRIMARY KEY (`id`)
+    ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+    
+    --
+    --
+    
+    DROP TABLE IF EXISTS `ezsite_public_access`;
+    
+    CREATE TABLE `ezsite_public_access` (
+      `public_access_identifier` varchar(255) COLLATE utf8mb4_unicode_520_ci NOT NULL,
+      `site_id` int(11) NOT NULL,
+      `site_access_group` varchar(255) COLLATE utf8mb4_unicode_520_ci NOT NULL DEFAULT '',
+      `status` int(11) NOT NULL,
+      `config` text COLLATE utf8mb4_unicode_520_ci NOT NULL,
+      `site_matcher_host` varchar(255) COLLATE utf8mb4_unicode_520_ci DEFAULT NULL,
+      PRIMARY KEY (`public_access_identifier`),
+      KEY `ezsite_public_access_site_id` (`site_id`),
+      CONSTRAINT `fk_ezsite_public_access_site_id` FOREIGN KEY (`site_id`) REFERENCES `ezsite` (`id`)
+    ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+    ```
+
+??? "Table structure for table `ezsite` and `ezsite_public_access` (PostgreSQL)"
+
+    ```sql
+    DROP TABLE IF EXISTS ezsite;
+    
+    CREATE SEQUENCE ezsite_seq;
+    
+    CREATE TABLE ezsite (
+      id int NOT NULL DEFAULT NEXTVAL ('ezsite_seq'),
+      name varchar(255) NOT NULL DEFAULT '',
+      created int NOT NULL,
+      PRIMARY KEY (id)
+    ) ;
+    
+    --
+    --
+    
+    DROP TABLE IF EXISTS ezsite_public_access;
+    
+    CREATE TABLE ezsite_public_access (
+      public_access_identifier varchar(255) NOT NULL,
+      site_id int NOT NULL,
+      site_access_group varchar(255) NOT NULL DEFAULT '',
+      status int NOT NULL,
+      config text NOT NULL,
+      site_matcher_host varchar(255) DEFAULT NULL,
+      PRIMARY KEY (public_access_identifier),
+      CONSTRAINT fk_ezsite_public_access_site_id FOREIGN KEY (site_id) REFERENCES ezsite (id)
+    ) ;
+    
+    CREATE INDEX ezsite_public_access_site_id ON ezsite_public_access (site_id);
+    ```
+
+### Update Apache/Nginx configuration
+
+Make sure that you Apache/Nginx configuration is up to date with Symfony 5.
+Refer to [the provided `vhost.template`](https://github.com/ezsystems/ezplatform/blob/master/doc/apache2/vhost.template)
+for an example.
+
+## 5\. Update the database
```

Comment:
Now I see that this section: https://github.com/ezsystems/developer-documentation/pull/953/files?short_path=919ebf3#diff-877068fcb53101273f10ca25b5750738R439 should be probably removed?